recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/images/core-image-thin-initramfs.bbappend

```diff
@@ -5,6 +5,8 @@ IMAGE_INSTALL_append = " \
     xen-xencommons \
     xen-xenstat \
     guest-addons \
+    guest-addons-rundoma \
```

Comment:
to be consistent, I would call it "-run-domX"

Reply:
Do you mean rundoma or both, together with rundomd?